lib/storage/metadata/mongoclient/MongoClientInterface.ts

```diff
@@ -2585,15 +2588,15 @@ class MongoClientInterface {
                     bucketInfos,
                 });
             })).catch(err => {
-                log.error('could not get list of collections', {
-                    method: 'getBucketInfos',
-                    error: err,
-                });
-                if (err && err instanceof ArsenalError) {
-                    return cb(err);
-                }
-                return cb(errors.InternalError);
+            log.error('could not get list of collections', {
+                method: 'getBucketInfos',
+                error: err,
             });
+            if (err && err instanceof ArsenalError) {
+                return cb(err);
+            }
+            return cb(errors.InternalError);
+        });
     }
```

Comment:
bad indent
```suggestion
                log.error('could not get list of collections', {
                    method: 'getBucketInfos',
                    error: err,
                });
                if (err && err instanceof ArsenalError) {
                    return cb(err);
                }
                return cb(errors.InternalError);
            });
    }
```

or wrap `).catch(err => {` to better show that there are 2 "nested" promises:
```suggestion
            })
        ).catch(err => {
            log.error('could not get list of collections', {
                method: 'getBucketInfos',
                error: err,
            });
            if (err && err instanceof ArsenalError) {
                return cb(err);
            }
            return cb(errors.InternalError);
        });
```